Lib/idlelib/editor.py

```diff
@@ -446,6 +446,26 @@ def set_line_and_column(self, event=None):
         self.status_bar.set_label('column', 'Col: %s' % column)
         self.status_bar.set_label('line', 'Ln: %s' % line)
 
+
+    """ Menu definitions and functions.
+    * self.menubar - the always visible horizontal menu bar.
+    * mainmenu.menudefs - a list of tuples, one for each menubar item.
+      Each tuple pairs a lower-case name and list of dropdown items.
+      Each item is a name, virtual event pair or None for separator.
+    * mainmenu.default_keydefs - maps events to keys.
+    * text.keydefs - same.
+    * cls.menu_specs - menubar name, titlecase display form pairs
+      with Alt-hotkey indicator.  A subset of menudefs items.
+    * self.menudict - map menu name to dropdown menu.
+    * self.recent_files_menu - 2nd level cascade in the file cascade.
+    * self.wmenu_end - set in __init__ (purpose unclear).
+
+    createmenubar, postwindowsmenu, update_menu_label, update_menu_state,
+    ApplyKeybings (2nd part), reset_help_menu_entries,
```

Comment:
```suggestion
    ApplyKeybindings (2nd part), reset_help_menu_entries,
```

Reply:
Thank you for catching this.  This diff is exactly the editor diff in #3669.  I am merging to main as is and will update the PR to remove its editor changes.  I will then edit the editer docstrings and comments in a new PR and will make this change there.